config/initializers/money.rb

```diff
@@ -1,3 +1,12 @@
+# This MoneyRails::ActionViewExtension module should get loaded
+# automatically when ActionView gets loaded, but very rarely when
+# setting up the app locally ItemHelper is loaded beforehand which
+# try to include this module and raises an uninitialized constant error.
+#
+# See: https://github.com/RubyMoney/money-rails/issues/614
+money_gem_dir = Gem::Specification.find_by_name("money-rails").gem_dir
+require "#{money_gem_dir}/lib/money-rails/helpers/action_view_extension"
```

Comment:
This is a bit sideways - you should be able to do this with one line:

```
require "money-rails/helpers/action_view_extension"
```

Reply:
Good point. I don't really understand how files are looked up for `require`.

Reply:
Fixed.